### PR TITLE
[plugin.audio.intergalacticfm] 3.0.0

### PR DESCRIPTION
**Pull request by PanderMusubi**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by PanderMusubi** [comment]
@basrieter could you have a look at this? We had this plugin in earlier versions, but the PRs were not created by me. Probably the review will be an easy one. Thanks.

File: plugin.audio.intergalacticfm/addon.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ]]></description>
+    <assets>
+        <icon>resources/icon.png</icon>
+        <fanart>resources/fanart.jpg</fanart>
+        <screenshot>screenshots/s1.png</screenshot>
+        <screenshot>screenshots/s2.png</screenshot>
+        <screenshot>screenshots/s3.png</screenshot>    
+    </assets>
+    <news>Updated with latest artwork metadata</news>
+    <license>The GNU General Public License (GPL)</license>
```
Comment (basrieter) [inline plugin.audio.intergalacticfm/addon.xml:38]: Please use the SPDX identifier here. See https://spdx.org/licenses/

File: plugin.audio.intergalacticfm/resources/streams.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{
```
Comment (basrieter) [inline plugin.audio.intergalacticfm/resources/streams.json:1]: Are these URL's publicly available? If so, where?

**Review by basrieter — CHANGES_REQUESTED** [review]
Please check my comments and make the Add-on checker happy.

File: plugin.audio.intergalacticfm/resources/streams.json (hunk 2 above)
Comment (PanderMusubi) [reply plugin.audio.intergalacticfm/resources/streams.json:1]: Yes at https://www.intergalactic.fm/ and under the header RADIO are three buttons. Each lead to a page that lists above the inline audio player ALTERNATIVE TUNE-IN and after that these URLs are listed. Does that answer the question?

Comment (PanderMusubi) [reply plugin.audio.intergalacticfm/resources/streams.json:1]: Yes at https://www.intergalactic.fm/ and under the header RADIO are three buttons. Each lead to a page that lists above the inline audio player ALTERNATIVE TUNE-IN and after that these URLs are listed. Does that answer the question? Does this need to go into the README?

Comment (PanderMusubi) [reply plugin.audio.intergalacticfm/resources/streams.json:1]: resolved in latest push

File: plugin.audio.intergalacticfm/addon.xml (hunk 1 above)
Comment (PanderMusubi) [reply plugin.audio.intergalacticfm/addon.xml:38]: resolved in latest push

**Comment by PanderMusubi** [comment]
@basrieter I try to update the PR here but get after following the update addon instructions this:

    $ git pull --strategy subtree --squash plugin.audio.intergalacticfm master
    From github.com:intergalacticfm/plugin.audio.intergalacticfm
     * branch                master     -> FETCH_HEAD
    hint: Pulling without specifying how to reconcile divergent branches is
    hint: discouraged. You can squelch this message by running one of the following
    hint: commands sometime before your next pull:
    hint: 
    hint:   git config pull.rebase false  # merge (the default strategy)
    hint:   git config pull.rebase true   # rebase
    hint:   git config pull.ff only       # fast-forward only
    hint: 
    hint: You can replace "git config" with "git config --global" to set a default
    hint: preference for all repositories. You can also pass --rebase, --no-rebase,
    hint: or --ff-only on the command line to override the configured default per
    hint: invocation.
    fatal: refusing to merge unrelated histories

How to continue?

**Comment by basrieter** [comment]
Can't you just push the new commits to your original PR branch?

**Comment by PanderMusubi** [comment]
Do you mean manually copy the changes into my PR branch and commit and push then?

**Comment by basrieter** [comment]
Try not to squash and just push the changes. It should update the PR.

**Pull request closed without merging**